docs/apppromotepipelinerun.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: promotepipelinerun
+spec:
+  resources:
+  - name: git-source
+    resourceRef:
+      name: git-app-repo
+  - name: docker-image
+    resourceRef:
+      name: docker-app-image
+  params:
+  - name: github-secret
+    value: promotesecret
+  - name: github-config
+    value: promoteconfigmap
+  - name: to
+    value: "https://github.com/akihikokuroda/gitops-prod.git"
```

Comment:
Should this be replaced with something else?

Reply:
yes, I fix it.